packages/ui/core-components/src/lib/unsorted/ui/Card.svelte

```diff
@@ -0,0 +1,14 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	export let className = '';
```

Comment:
`class` would be consistent with the accordion. See: https://docs.evidence.dev/components/accordion/#props-class

```suggestion
	let className = undefined;
	export { className as class };
```